geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/QueryPlanner.scala

```diff
@@ -376,12 +379,13 @@ object QueryPlanner extends Logging {
     val sft = if (query.getHints.isBinQuery) {
       BinAggregatingIterator.BIN_SFT
     } else if (query.getHints.isDensityQuery) {
-      Z3DensityIterator.DENSITY_SFT
-    } else if (query.getHints.containsKey(TEMPORAL_DENSITY_KEY)) {
+      KryoLazyDensityIterator.DENSITY_SFT
+    } else if (query.getHints.isStatsIteratorQuery) {
+      KryoLazyStatsIterator.createFeatureType(baseSft)
+    } else if (query.getHints.isTemporalDensityQuery) {
       TemporalDensityIterator.createFeatureType(baseSft)
     } else if (query.getHints.containsKey(MAP_AGGREGATION_KEY)) {
```

Comment:
Should use query.getHints.isMapAggregatingQuery - I had changed it in the base PR.